server.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const Concorda = require('./lib/index')
+
+var Hapi = require('hapi')
+var Bell = require('bell')
+var Chairo = require('chairo')
+var Cookie = require('hapi-auth-cookie')
+
+module.exports = function (options, done) {
+  // Create our server.
+  var server = new Hapi.Server({debug: {request: ['error']}})
+  server.connection({port: options.server.port})
+
+  // Declare our Hapi plugin list.
+  var plugins = [
+    {register: Bell},
+    {register: Cookie},
+    {register: Chairo, options: options.chairo}
+  ]
+
+  // Register our plugins.
+  server.register(plugins, function (err) {
+    if (err) {
+      return done(err)
+    }
+
+    server.seneca.use(Concorda, options)
+
+    // Kick off the server.
+    server.start(function (err) {
```

Comment:
this is still happening in the server.js file. it needs to happen in the start.js.